# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/cairo/cairo/src/cairo-backend-private.h
# Commit: 3cdfb39376b7
# Full Hash: 3cdfb39376b736597fe658a322389117686b7e82
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-05-07 09:13:07
# Regressor Bug: 1892913
# File Overlap Count: 3
# Description:
#   Bug 1892913 - patch 1 - Update cairo source to 1.18.0 r=gfx-reviewers,lsalzman
#   
#   Wholesale replacement of files in gfx/cairo/cairo with contents of the 1.18.0 release tarball,
#   omitting subdirectories that are not part of the core library build.
#   
# ==============================================================================

diff -r 9daa351f875b -r 3cdfb39376b7 gfx/cairo/cairo/src/cairo-backend-private.h
--- a/gfx/cairo/cairo/src/cairo-backend-private.h	Mon May 06 19:22:35 2024 +0000
+++ b/gfx/cairo/cairo/src/cairo-backend-private.h	Mon May 06 19:25:12 2024 +0000
@@ -68,6 +68,7 @@
     cairo_status_t (*set_line_cap) (void *cr, cairo_line_cap_t line_cap);
     cairo_status_t (*set_line_join) (void *cr, cairo_line_join_t line_join);
     cairo_status_t (*set_line_width) (void *cr, double line_width);
+    cairo_status_t (*set_hairline) (void *cr, cairo_bool_t set_hairline);
     cairo_status_t (*set_miter_limit) (void *cr, double limit);
     cairo_status_t (*set_opacity) (void *cr, double opacity);
     cairo_status_t (*set_operator) (void *cr, cairo_operator_t op);
@@ -79,6 +80,7 @@
     cairo_line_cap_t (*get_line_cap) (void *cr);
     cairo_line_join_t (*get_line_join) (void *cr);
     double (*get_line_width) (void *cr);
+    cairo_bool_t (*get_hairline) (void *cr);
     double (*get_miter_limit) (void *cr);
     double (*get_opacity) (void *cr);
     cairo_operator_t (*get_operator) (void *cr);